pkg/tests/prow_operator_tests.go

```diff
@@ -23,14 +24,17 @@ var _ = ginkgo.Describe("Prow Operator Tests", func() {
 		apiextensions, err := clientset.NewForConfig(config)
 		Expect(err).NotTo(HaveOccurred())
 
-		// Make sure the CRD exists
-		result, err := apiextensions.ApiextensionsV1().CustomResourceDefinitions().Get("addons.addons.managed.openshift.io", v1.GetOptions{})
+
+		// Make sure the CRD exists. Using crd name for reference-addon used in the example.
+		crdName := "referenceaddons.reference.addons.managed.openshift.io"
```

Comment:
I understand why we'd want to put a variable name instead of the direct string, but in this case I'd move it up so it's easier to find.